be/src/storage/rowset/column_reader.h

```diff
@@ -70,16 +70,6 @@ class ParsedPage;
 class ZoneMapIndexPB;
 class ZoneMapPB;
 
-struct ColumnReaderOptions {
```

Comment:
Why remove this?
I think we may add new options in the future.

Reply:
There is no way to ensure all references will be updated after adding a new field in this struct, which is error-prone.